### ISSUE TITLE
What does stage 5 of potion brewing do?

### ISSUE THREAD

**Issue by RasmusKD** [mcMMO-Dev/mcMMO#4627]
"represents a base potion with one ingredient where the amplifiers are swapped"

But what is counted as amplifiers? if this refers to glowstone and redstone, you haven't been able to swap them since 1.9

**Comment by Geekerandy** [comment]
Yes, that’s exactly what they refer to, nossr just hasn’t removed the code because it’s not his own and he dislikes messing with Alchemy to fix it
